components/odh-notebook-controller/Makefile

```diff
@@ -87,7 +87,14 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 .PHONY: test
-test: manifests generate fmt vet envtest ## Run tests.
+test: test-with-rbac-false test-with-rbac-true
+test-with-rbac-false: manifests generate fmt vet envtest ## Run tests.
+	@export SET_PIPELINE_RBAC=false && \
```

Comment:
Why add the `@`? When I want to run the tests in debugger, I like to copy out the test command from `make test` and paste it into IDE. WIth @ the command is no longer printed.

Reply:
Removed that @, and added 
`fmt.Printf("SET_PIPELINE_RBAC is: %s\n", os.Getenv("SET_PIPELINE_RBAC"))` to debug the value